app/views/members/_search.html.erb

```diff
@@ -0,0 +1,4 @@
+<%= form_tag(members_path, method: :get) do %>
+  <%= text_field_tag "query", search_params %>
```

Comment:
There are double quotes around `query` here but single quotes around `Search` on the line below. Please use double quotes consistently here as they match what is used for HTML attributes.